README.md

```diff
@@ -175,8 +175,8 @@ The intention is that MiniRust has no more UB than that, but it *does* have less
 
 - It is *not* UB to dereference a null, unaligned, or dangling raw pointer. In other words, `addr_of!(*ptr)` is always defined.
   However, if the `*ptr` place expression is being offset, that still needs to happen in-bounds, and actual loads/stores need to be sufficiently aligned.
-- It is *not* always UB to create a reference or `Box` to an invalid value, or one that is dangling.
-  However, it *is* UB to create a reference or `Box` to an *uninhabited type*, or one that is null or unaligned.
+- It is *not* always UB to create a reference or `Box` to an invalid value.
+  However, it *is* UB to create a reference or `Box` to an *uninhabited type*, or one that is dangling or unaligned.
   Moreover, when evaluating an `&[mut]` value expression, dangling references are UB.
   (Dangling references nested in fields will likely also become UB when the aliasing model is added.)
```

Comment:
We changed this in #244, but I am somewhat unsure about how to update this text.